content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -2,4 +2,940 @@
+However, before we do this, let's add another describe block after all the other
+tests inside the "LoginForm" describe block.
+
+Update your spec file, adding a new "form tests" describe block after all the
+tests in the "LoginForm" block like so, and save it:
+
+```js
+describe('LoginForm', () => {
+  // existing tests
+
+  describe('form tests', () => {
+    // nothing here yet
+  })
+})
+```
+
+The Cypress app should update to show the new describe block, but there won't be
+anything inside it, because we haven't written any tests yet.
+
+## Testing user interactions
+
+intro
+
+### Focusing tests
+
+In spec files with many tests, it can be helpful to "focus" a test while writing
+it, so only that test is run and shown in the Cypress app. In order to do so,
+just update your test to use `it.only()` instead of `it()`. Describe blocks can
+also be focused by using `describe.only()` to run only certain groups of tests.
+
+Focusing tests and saving your spec files as you're adding tests and assertions
+can help you catch errors early, which will make debugging easier. Just be sure
+to remove any `.only`s you've added after you've finished writing your tests, so
+the other tests are shown again!
+
+See the guide on
+[Excluding and Including Tests](/guides/core-concepts/writing-and-organizing-tests#Excluding-and-Including-Tests)
+for more examples.
+
+### Stubbing functions
+
+Many components accept function props, which allows the component to notify its
+parent whenever a particular event occurs. For example, it may be necessary for
+a parent component to pass an `onClick` function, also known as a "callback" or
+"handler," into a child `Button` component to allow the parent to do something
+whenever the user clicks the button.
+
+In order to allow testing components outside their parent, we'll use the
+[`cy.spy()`](/api/commands/spy) command, which creates a function that can be
+passed into a component as a prop, which can be asserted on using
+[`.should()`](/api/commands/should) to see that it was called (or not), or
+called with specific arguments.
+
+### Testing form submit
+
+The next test is going to be a bit more involved than our previous tests,
+because we're going to be entering text into form fields, submitting the form,
+and verifying that a callback is getting called.
+
+First, let's create the new test. Inside the "form tests" block, add this test
+(which we're focusing with `.only` as described above) and save the spec file:
+
+```js
+it.only('should call onLogin with username and password on login', () => {
+  // nothing here yet
+})
+```
+
+Inside the test, we first need to create a spy that we can pass into the
+`LoginForm` component. We're also going to need to refer to the spy later, to
+assert that it was called correctly, so we'll use [`.as()`](/api/commands/as) to
+assign the `onLoginSpy` alias for later use and to make it easier to see the spy
+in the Cypress app.
+
+(callout: aliases ?)
```

Comment:
I think a reference to alias is a good idea

Reply:
Great feedback - I've addressed all of your comments around this section in 1f7526579627e2e1219af36d49f6bb2d80b4d02a